src/flask/blueprints.py

```diff
@@ -272,11 +272,7 @@ def __init__(
         self.url_prefix = url_prefix
         self.subdomain = subdomain
         self.deferred_functions: t.List[DeferredSetupFunction] = []
-
-        if url_defaults is None:
-            url_defaults = {}
-
-        self.url_values_defaults = url_defaults
+        self.url_values_defaults = {} if url_defaults is None else url_defaults
```

Comment:
This code fix does not seem to change anything.

Reply:
Yes, But seems in single line. As python provide one line if else

Reply:
Both options are perfectly valid, please do not send PRs with such minor changes. They are more noise than useful.